Right to be Forgotten (#294)
ietf-wg-scitt/draft-ietf-scitt-architecture: draft-ietf-scitt-architecture.md

Transparency Services can have an authorization policy controlling who can access the Append-only Log.
While this can be used to limit who can read the Log, it may also limit the usefulness of the system.

> Once a Signed Statement is inserted into the Append-only Log maintained by a Transparency Service, it cannot be removed from the Log.

Reviewer: ```suggestion
Once a Signed Statement is secured by the Append-only Log maintained by a Transparency Service, it cannot be removed from the Log.
```

> Once a Signed Statement is registered on an Append-only Log maintained by a Transparency Service, the fact that a Signed Statement was registered cannot be removed from the Append-only Log.

Reviewer 2: ```suggestion
Once a cryptographic digest of the Signed Statement is registered on the Append-only Log, the registration cannot be removed from the Append-only Log.
```

Reviewer 2: The narrowed text in this section is better than what's currently in the draft. However, this section is problematic as it makes it appear the actual content of the signed statement is included in the log, which could detract users from registering important content. We need better text for this section as the original draft suggested the content of the Signed Statement was recorded, making this more problematic and concerning for PII.

Reviewer 3: > The narrowed text in this section is better than what's currently in the draft. However, this section is problematic as it makes it appear the actual content of the signed statement is included in the log, which could detract users from registering important content.

Where in the architecture document does it mandate (MUST) that the payload of the Signed Statement is a hash and locator, not the content itself? I understand your perspective on this, and agree it is a better use of the TS for efficiency, but I thought the architecture _allows_ for either, and I think there are use cases where that is acceptable or preferred (instead of external looks up to external systems, even if in a properly secured network; our own uses cases document has examples with air-gapped networks).
